components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -18,6 +18,10 @@
 /** Minimum (typical) height of a Material Tab bar. */
 static const CGFloat kMinHeight = 48;
 
+static NSString *const kImageString = @"image";
+static NSString *const kSelectedImageString = @"selectedImage";
+static NSString *const kTitleString = @"title";
```

Comment:
Please use “KeyPath” as the suffix instead of “String” for these variables.

Reply:
Sure. These are taken from Bottom Navigation (all the code is), but I can deviate from the team's norms.

Reply:
Done.

Reply:
Naming variables with `String` suffices is definitely not a team norm; any code that's written as such is out of line with Objective-C style guidelines.